<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkWindow</class>
 <widget class="QMainWindow" name="SocialNetworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>135</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="loginEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="profileLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="friendsTable">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="friendsLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Friends</string>
    </property>
   </widget>
   <widget class="QLabel" name="postsLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Posts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFriendButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Friend</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="friendSuggestionsTable">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>350</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="friendSuggestionsLabel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Friend Suggestions</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please enter a valid name...</string>
    </property>
   </widget>
   <widget class="QLabel" name="post">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="signupButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="signupLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="signupNameLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="signupYearLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Year of Birth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="signupZipLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zip Code:</string>
    </property>
   </widget>
   <widget class="QLabel" name="signupFriendsLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Friend IDs:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="signupNameEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="signupYearEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="signupZipEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="signupFriendsEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>330</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="signupOKButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="signupErrorLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please enter proper values!!!!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="writePostButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string> Write Post</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="writePostEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="signupbackButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="profilePic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>&lt;PFP&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="signupPfp">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>                &lt;PFP&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signupUploadButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload a Picture</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="searchEdit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="searchLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search by ID:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>35</y>
      <width>31</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>üîç</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
